src/serial.rs

```diff
@@ -74,129 +49,41 @@ pub unsafe trait RxPin<USART> {}
 unsafe impl TxPin<USART1> for gpioa::PA9<AF7> {}
 unsafe impl TxPin<USART1> for gpiob::PB6<AF7> {}
 unsafe impl TxPin<USART1> for gpioc::PC4<AF7> {}
-#[cfg(any(
-    feature = "stm32f302",
-    feature = "stm32f303xb",
-    feature = "stm32f303xc",
-    feature = "stm32f303xd",
-    feature = "stm32f303xe",
-    feature = "stm32f373",
-    feature = "stm32f378",
-    feature = "stm32f328",
-    feature = "stm32f358",
-    feature = "stm32f398"
-))]
-unsafe impl TxPin<USART1> for gpioe::PE0<AF7> {}
-
 unsafe impl RxPin<USART1> for gpioa::PA10<AF7> {}
 unsafe impl RxPin<USART1> for gpiob::PB7<AF7> {}
 unsafe impl RxPin<USART1> for gpioc::PC5<AF7> {}
-#[cfg(any(
-    feature = "stm32f302",
-    feature = "stm32f303xb",
-    feature = "stm32f303xc",
-    feature = "stm32f303xd",
-    feature = "stm32f303xe",
-    feature = "stm32f373",
-    feature = "stm32f378",
-    feature = "stm32f328",
-    feature = "stm32f358",
-    feature = "stm32f398"
-))]
-unsafe impl RxPin<USART1> for gpioe::PE1<AF7> {}
 
 unsafe impl TxPin<USART2> for gpioa::PA2<AF7> {}
-// unsafe impl TxPin<USART2> for gpioa::PA14<AF7> {}
-// unsafe impl TxPin<USART2> for gpiob::PB3<AF7> {}
```

Comment:
I don't know why these pins were commented out. I just added them in again, with the features derived from the available AF mappings according to the CubeMX DB.